styleguide/derek/examples/_office365.html

```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Overview - Features -->
+   <div class="container-fluid feature-container bg-crumpled-paper river">
+
+      <div class="row">
+        <div class="col-md-12 center-block">
+          <h2 class="center">Millions of users are choosing Office 365.<br> Find out why.</h2>
+          <p class="center">Power productivity, inspire collaboration from anywhere, <br>on any device with Office 365.</p>
+        </div>
+      </div>
+
+      <div class="row">
+        <!-- Left Column -->
+        <div class="col-md-5">
+          <!-- Feature 1 -->
+          <div class="col-md-12 feature">
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb idio-high-five black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Work Better Together</h3>
+              <p>Collaborate easier with access to email, calendar, files and Offfice applications from any device, on the road, at home or at the office.</p>
+            </div>
+          </div>
+          <!-- Feature 2 -->
+          <div class="col-md-12 feature">
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb auto-process black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Never Update Again</h3>
+              <p>Always enjoy access to the latest versions, feature releases and security updates of Microsoft Office, Outlook, SharePoint, and more. <a href="javascript:">Office 2016 now available.</a></p>
+            </div>
+          </div>
+          <!-- Feature 3 -->
+          <div class="col-md-12 feature">
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb gen-locked-checked-shield black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Work More Securely</h3>
+              <p>Work and share files confidently with built-in, customizable security and privacy controls that help keep your critical data safe.</p>
+            </div>
+          </div>
+          <!-- CTA -->
+          <div class="row half-padding">
+              <div class="col-md-9 col-md-offset-2">
+                <a href="#" class="button learning center-block">View Plans and Pricing</a>
```

Comment:
center-block shouldnt be needed here as the buttons will span the full width of the container.